GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -319,7 +319,10 @@ private void ReloadCommitInfo()
 
             if (_revision is not null && !_revision.IsArtificial)
             {
-                StartAsyncDataLoad(Module.EffectiveSettings);
+                if (Module.GetEffectiveSettings() is DistributedSettings distributedSettings)
+                {
+                    StartAsyncDataLoad(distributedSettings);
+                }
```

Comment:
No error hiding, please!

```suggestion
                StartAsyncDataLoad((DistributedSettings )Module.GetEffectiveSettings());
```

Reply:
If a bug creeps in, your proposal will make the app unusable. I'll add an assert instead.